lib/model/app_model/list_invoices.dart

```diff
@@ -9,80 +10,88 @@ class AppListInvoices {
   factory AppListInvoices.fromJSON(Map<String, dynamic> json,
       {bool snackCase = false, bool isObject = false, String? status}) {
     var invoices = json.withKey("invoices", snackCase: snackCase) as List;
+    List<AppInvoice> list = [];
+
     if (invoices.isNotEmpty) {
-      var appInvoices = invoices.map((invoice) {
-        var temp = AppInvoice.fromJSON(invoice,
+      for (int i = 0; i < invoices.length; i++) {
```

Comment:
```suggestion
      for (var invoice in invoices) {
```
this syntax will not work?